### PR TITLE
Add pdJ for formatted JSON disassembly

### PR DESCRIPTION
**Pull request by thestr4ng3r**
What I described in https://github.com/radareorg/cutter/issues/177

I called it `pdJ` now, but of course we can still discuss the name.

Tests: https://github.com/radare/radare2-regressions/pull/1096

### CODE REVIEW

File: libr/core/disasm.c [hunk 1]
```diff
@@ -3783,14 +3849,45 @@ static void ds_print_comments_right(RDisasmState *ds) {
...
+					if (ds->use_json) {
+						comment = escstr = ds_esc_str (ds, comment, (int)strlen (comment), NULL);
+					}
+
+					if (comment) {
+						r_cons_printf ("; %s", comment);
+					}
+
+					if (escstr) {
+						free (escstr);
```
Comment (radare) [inline libr/core/disasm.c]: Free already checks for null, drop this if

File: libr/core/disasm.c [hunk 2]
```diff
@@ -3783,14 +3849,45 @@ static void ds_print_comments_right(RDisasmState *ds) {
 					r_cons_strcat (ds->color_usrcmt);
 				}
 				if (strchr (comment, '\n')) {
-					r_cons_newline ();
 					char *align = r_str_newf ("%s  ;^ ", ds->pre);
-					char *c = r_str_prefix_all (strdup (comment), align);
-					r_cons_strcat (c);
+					comment = strdup (comment);
```
Comment (radare) [inline libr/core/disasm.c]: Maybe chk for null?

**Comment by radare** [comment]
Why its red?

File: libr/core/disasm.c (hunk 2 above)
Comment (thestr4ng3r) [reply libr/core/disasm.c]: done

**Comment by thestr4ng3r** [comment]
You mean appveyor? Probably related to #9003 and #8690.

**Comment by Maijin** [comment]
@radare red is not related to this PR, can we merge it ?

**Pull request merged**